apps/lfx-one/src/app/modules/dashboards/executive-director/components/engaged-community-drawer/engaged-community-drawer.component.html

```
@@ -155,5 +155,18 @@ <h3 class="text-sm font-semibold text-gray-900">Community Breakdown</h3>
         <lfx-chart type="bar" [data]="breakdownChartData()" [options]="breakdownChartOptions" height="100%"></lfx-chart>
       </div>
     </div>
+
+    <!-- Weekly Sessions Trend -->
+    @if (brandReachData().dailyTrend.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)